src/code_807CD9C.c

```diff
@@ -71,11 +70,11 @@ void sub_807CD9C(Entity *pokemon, Entity *target, u32 direction)
     else
     {
         if (pokemon == target) {
-            SetMessageArgument(gAvailablePokemonNames,target,0);
+            SetMessageArgument(gFormatBuffer_Monsters[0],target,0);
```

Comment:
There needs to be a lot of work done, not in this PR, but properly renaming all the functions that write to the format buffers.
You can't trust the name of the buffer itself to know what's being written, you have to name the functions appropriately, in that case.
This one should be PrintEntityName, SetMessageArgument_2 should be PrintMonsterName, and so on.

(Maybe I can do that once this gets merged, seems like a fun reversing project)

Reply:
Yeah, it's definitely a good idea to do so. But as you said, let's keep it for another PR.